src/main.zig

```diff
@@ -518,15 +518,6 @@ pub fn main() void { // MARK: main()
 	file_monitor.init();
 	defer file_monitor.deinit();
 
-	Window.init();
```

Comment:
Now please for every other thing that you reordered, double check all implementations if the reordering broke anything, or move them back to the original order.

Reply:
I reverted all of the reordering besides putting the `gui.openWindow("change_name")` call after `server.terrain.globalInit` and putting `settings.launchConfig.init();` before `gui.initWindowList`